requirements-dev.txt

```diff
@@ -0,0 +1,8 @@
+tox>=2.3.1
```

Comment:
why we removed  test_requirements.txt and added this file? this is about current naming convention standards?

Reply:
No special reason. Just because some tools like pyup.io use requirements * to check dependencies.
If you prefer we can go back as it was before.